src/platforms/android/common/proguard.mdx

```diff
@@ -127,39 +127,6 @@ sentry {
 
 Now when you build your app, the plugin will upload the ProGuard/R8 mappings, source bundle, and native symbols as you configured them to Sentry.
 
-### Multi-Dex Support
```

Comment:
already available via https://docs.sentry.io/platforms/android/configuration/multi-dex/

Reply:
Should we then maybe add a link here instead?

Reply:
no, multi-dex isn't proguard related